CONTRIBUTING.md

```diff
@@ -111,6 +111,10 @@ Mypy install and run.
 **Example: Invoke tox, breaking into the debugger on failure**
 `tox -e whl -c ../../../eng/tox/tox.ini -- --pdb`
 
+### Performance Testing
+
+The `azure-devtools` package now supports a custom performance testing framework. For full details on this framework, and how to write and run tests for an SDK - see the [perfstress tests documentation](https://github.com/Azure/azure-sdk-for-python/blob/master/tools/azure-devtools/doc/perfstress_tests.md).
```

Comment:
Can you do a more neutral setence without `azure-devtools`, I don't want people to feel there is a package they can get. Especially we don't use the one from PyPI anymore and it created confusion.

Something like "This repo test framework yadayadayadayada"